workspaces/spectacle/src/in-memory/index.ts

```diff
@@ -163,7 +164,13 @@ export class InMemoryCapturesService implements IOpticCapturesService {
 ////////////////////////////////////////////////////////////////////////////////
 
 export class InMemoryConfigRepository implements IOpticConfigRepository {
-  ignoreRequests: string[] = [];
+  addIgnoreRule(rule: string): Promise<void> {
+    return Promise.resolve(undefined);
```

Comment:
do you plan to make it work by maintaining an array here?

Reply:
it doesn't really need to 'work' this is only for persistence, and the contract of demo/testing entry points is that if you refresh, it resets